ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml

```diff
@@ -43,18 +43,20 @@ StatusSectionLayout {
     property bool hasAddedContacts: false
 
     readonly property string filteredSelectedTags: {
-        if (!community || !community.tags)
+        if (!community && !community.tags) {
+            try {
+                const json = JSON.parse(community.tags);
+                const tagsArray = json.map(tag => {
+                                               return tag.name;
+                                           });
+                return JSON.stringify(tagsArray);
+            }
+            catch (e) {
+                console.warn("Error parsing community tags: ", !!community.tags ? community.tags : "", " error: ", e.message)
```

Comment:
And if we're checking above for valid objects there is no reason to check here also.

Reply:
done

Reply:
@alexjba fyi just pushed one more change to remove
`Warning: Error parsing community tags:  null  error:  Cannot read property 'map' of null (qrc:/app/AppLayouts/Chat/views/CommunitySettingsView.qml:55, expression for filteredSelectedTags)`

Reply:
I don't understand how is it entering the if statement with null community.tags if we're checking it for null 🤯

Reply:
Because `null!==undefined`? :)

Reply:
yeah, but both undefined and null should be falsy values.